sentry-kotlin-multiplatform/build.gradle.kts

```diff
@@ -25,8 +25,7 @@ kotlin {
     android {
         publishAllLibraryVariants()
     }
-
-    jvm()
+    //jvm()
```

Comment:
Why is it commented?

Reply:
the expect/actual keywords force me to write implementations for every specified target and it's much easier to focus on iOS and Android if the compiler doesn't send out an error all the time for missing implementations.

Also since there is a lot of overlap between JVM and Android sdks I want to explore a way to reduce code duplication in another issue if there's time left